route/account/_account-id/token/index.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+module.exports = (server, path, options) => {
+    return options.container.call((readBodyBufferMiddleware, tokenManager, validateRequestQueryMiddleware, validateSignatureMiddleware) => {
+        return {
+            name: "account-token-create",
+            post: [
+                readBodyBufferMiddleware(),
+                validateSignatureMiddleware(),
+                validateRequestQueryMiddleware("/account/token-create-request.json"),
+                (req, res, next) => {
+                    var publicFlag;
+
+                    publicFlag = false;
+
+                    // Query string values are always strings - compare as such
+                    if (req.query && req.query.public === "true") {
```

Comment:
Are we sure we want to compare to the string? I know we validate by type as well, but maybe we should do `req.query.public.toLowercase() === "true"` as least, just to make sure somehow we don't get `TruE`.